sos/plugins/docker.py

```diff
@@ -29,20 +29,47 @@ class Docker(Plugin):
         ("all", "enable capture for all containers, even containers "
             "that have terminated", 'fast', False),
         ("logs", "capture logs for running containers",
-            'fast', False)
+            'fast', False),
+        ("size", "capture image sizes for ps", 'slow', False)
```

Comment:
I think we could use `'docker ps'` in here - for people who aren't familiar with the nits and bits it's potentially confusing with the `procps` command of the same name.